bibliopixel/drivers/SPI/errors.py

```diff
@@ -0,0 +1,35 @@
+PERMISSION_ERROR = """Cannot access SPI device.
+Please see
```

Comment:
NEFR... I didn't write this :P All I did was extract them into errors.py
I will change this in another review.